tests/scenario_certification/test_criticality_summary.py

```
@@ -0,0 +1,575 @@
+        horizon=80,
+        dt=0.1,
+        seed_limit=1,
+        materialization={},
+        planner_runs={},
+        pair_summary={
+            "pairs": 0,
+            "row_status_counts": _zero_row_status_counts(),
+            "mean_deltas_completed_pairs": {},
+        },
+    )
+    with pytest.raises(Exception):
```

Ruff (0.15.14)

[warning] Do not assert blind exception: `Exception`

(B017)